workerapi/client/commands.go

```diff
@@ -3,46 +3,42 @@ package client
 import (
 	"log"
 	"strings"
+	"time"
 
 	"github.com/luci/go-render/render"
-	"github.com/scootdev/scoot/workerapi/gen-go/worker"
+	"github.com/scootdev/scoot/runner"
 	"github.com/spf13/cobra"
 )
 
 // Run
 var runArgv *string
-var runcmd *worker.RunCommand = worker.NewRunCommand()
+var runcmd *runner.Command = &runner.Command{}
 
-func makeRunCmd(c *Client) *cobra.Command {
+func makeRunCmd(c *cliClient) *cobra.Command {
 	r := &cobra.Command{
 		Use:   "run",
 		Short: "runs a command",
 		RunE:  c.run,
 	}
 	r.Flags().StringVar(&c.addr, "addr", "localhost:9090", "address to connect to")
 	runArgv = r.Flags().String("argv", "", "comma separated list of binary and args")
-	runcmd.SnapshotId = r.Flags().String("snapshotid", "", "snapshot/patch id.")
-	runcmd.TimeoutMs = r.Flags().Int32("timeout_ms", 0, "timeout before aborting cmd")
+	runcmd.SnapshotId = *r.Flags().String("snapshotid", "", "snapshot/patch id.")
+	runcmd.Timeout = time.Duration(*r.Flags().Int32("timeout_ms", 0, "timeout to abort cmd")) * time.Millisecond
 	return r
 }
-func (c *Client) run(cmd *cobra.Command, args []string) error {
+func (c *cliClient) run(cmd *cobra.Command, args []string) error {
 	runcmd.Argv = strings.Split(*runArgv, ",")
 	log.Println("Calling run rpc to cloud worker", args, render.Render(runcmd))
 
-	client, err := c.Dial()
-	if err != nil {
-		return err
-	}
-
-	status, err := client.Run(runcmd)
+	status := c.Run(runcmd)
```

Comment:
Errors are incorporated into runner.ProcessStatus as came up in the previous PR. I only print it because I didn't necessarily see the value in returning it to cobra. I'm not 100% comfortable with my decision to remove err from a bunch of function signatures, so I can be convinced to add it back.